core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -80,20 +80,20 @@ <h4 class="bold-title">
                         </section>
                         <section class="col-lg-3 col-md-3 pad-left0">
                             <div class="radio-container">
-                                <input type="radio" name="runtime" id="java" [value]="java" [checked]="runtime == 'java'" (click)="onSelectionChange('java')">
-                                <label for="java"><span class='outer'></span><span class="background"></span>Java 8</label>
+                                <input type="radio" name="runtime" id="python2-7" [value]="python2-7" [checked]="runtime == 'python2.7'" (change)="onSelectionChange('python2.7')">
```

Comment:
Why are we adding`-` instead of`.` for runtime version?

Reply:
Jazz UI is failing to parse the HTML document when using . for runtime version.

Reply:
@supritat  Can we use `environment.oss.ts` to store all runtime values and use them here?